packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/create-local-file-node.js

```diff
@@ -230,13 +247,25 @@ export const createRemoteMediaItemNode = async ({
     maxFileSizeBytes,
   } = pluginOptions.type?.MediaItem?.localFile
 
+  const maybePrintWarning = maybePrintWarningFactory(reporter)
+
   // if this file is larger than maxFileSizeBytes, don't fetch the remote file
   if (fileSize > maxFileSizeBytes) {
+    maybePrintWarning({
+      message: `At least one remote media item not fetched because its size exceeded the maxFileSizeBytes config option. Other media items items may have been skipped for this reason as well but this warning is only logged once to avoid terminal spam.`,
```

Comment:
Can we allow this to log a few times and add the file name and/or db ID to the warning to help with debugging?

Reply:
Oo the word "items" is printed twice as well :)

Reply:
Totally! Does three times sound like the correct amount?

Reply:
I think we could do 10 without being too disruptive. Maybe try logging a bunch out to see what it looks like